docs/GettingStarted.md

```diff
@@ -274,6 +274,7 @@ Where `options` is an optional `Hash` that accepts the following parameters:
 | Key | Description | Default |
 | --- | --- | --- |
 | ``service_name`` | Service name used for `elasticsearch` instrumentation | elasticsearch |
+| ``quantize`` | Hash containing options for quantization. May include `:keep` with an Array of keys to not quantize, or `:exclude` with Array of keys to exclude entirely. | {} |
```

Comment:
Instead of `keep`, I would say to call it `show` since the value was already there with/without this config (it's just obfuscated or not). What do you think?

Also there is a way to say "show all fields" (aka disabling the quantizer/obfuscator)

Reply:
Hmmm, `show` is reasonable. Really orients the concept towards UI display, but that's not necessarily a bad thing. Only bad if they think not "showing" is "hiding", and that old data that wasn't "shown" can simply be revealed later by changing the setting, as opposed to it actually being permanently lost.

Reply:
I think we could probably add some setting for show all... maybe if instead of a hash, you pass `true`? or a symbol `:all`?